lib/ServerError.js

```diff
@@ -47,6 +47,11 @@ const ServerError = {
     });
   },
 
+  addressNotFound: request => {
+    logger.Debug(`Host name not found: ${JSON.stringify(request, null, 2)}`);
+    logger.Error(`Could not resolve hostname: ${request.cause.hostname}`, { hideTimestamp: true });
```

Comment:
How about including potential reasons why the host name was not found ? Like there can be typo in the URL or double check your internet connection ? Ideally we would check which one is it and display information how to resolve this problem ?

Reply:
https://github.com/sindresorhus/is-online :)

Reply:
BTW. Im for merging what we have (since it is clearly better) and extracting further work to another issue.